pkg/selinux/setroubleshoot.js

```diff
@@ -114,7 +114,7 @@ const initStore = function(rootElement) {
                     dataStore.entries[idx].details.pluginAnalysis[fixId].fix = {
                         plugin: analysisId,
                         running: false,
-                        result: error,
+                        result: error.toString(),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18213-20230120-084740-51c9ad03-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Hmm, good point -- let's do that properly, and sabotage `runcon` in the test, to simulate an error.